content/blog/2022/07/v1.1.1-release.md

```diff
@@ -0,0 +1,37 @@
+---
+title: v1.1.1 is out 🎉
+authors:
+- Víctor Cuadrado Juan
+date: 2022-07-25
+---
+
+We are happy to announce the first minor release of v1.0: **v1.1.1** is
+out!
+
+Apart from being a nice looking number, this new **v1.1.1** includes:
+
+* Improved the policies API for Sigstore verification, and added new
+  backwards-compatible WaPC host callback `v2/verify` functions to the API. You
+  can [check them out
+  here](https://docs.kubewarden.io/writing-policies/spec/host-capabilities/signature-verifier-policies)
+  if you want to add support for your language of choice.
+  
+  We have made use of them in the verify-image-signatures policy to simplify
+  verification of GitHub Actions signatures, and others. Read about it in [this
+  blog post](../verify-signatures-with-gha-and-prefix/).
+* Made Kubewarden resilient to hiccups in the upstream Sigstore's repository for
```

Comment:
This whole paragraph will need to be rephrased to first include what we have done rather than what are the benefits. This means, it will look something like this:
```suggestion
* Improved handling of metadata updates in `policy-sever` and `kwctl` v1.1.1. 
Not only does this makes Kubewarden more resilient to hiccups in the 
upstream Sigstore repositories for [Fulcio](https://github.com/sigstore/fulcio) and [Rekor](https://github.com/sigstore/rekor) but the user will be informed 
of the updates and they will continue to fail safely by issuing false negatives, if required.
```

Reply:
The idea with this paragraph was to first say what we did (made Kubewarden resilient to the Sigstore repository hiccups), and then explain why and what are the benefits. I'm not sure how to reword it better tbf, but following with the suggestion would mean changing the meaning of the information.

Reply:
I understand the angle you're coming from, but this is one of the points being covered under what v1.1.1 includes. If we start by saying what we did, that would be in conflict with the way we're trying to frame the point.

Reply:
So can we start off by saying what we improved and then list the benefits? Kubewarden becoming resilient is a benefit or an after-effect of the action we took. So flipping the sentence to ensure we convey the action first is what I wanted to do.

Reply:
Whoops! now I see it, sorry. Fixed it in https://github.com/kubewarden/kubewarden.io/pull/119/commits/54d8b63ad2f78ca9646e3eb8613f02a01a41766e, thanks!

Reply:
Thanks! But I don't see the changes. The second bullet point still starts off as "Made Kubewarden resilient to hiccups in the upstream Sigstore's repository for". We haven't yet mentioned what exactly did we do to make it resilient in the first part.

Reply:
Addressed in https://github.com/kubewarden/kubewarden.io/pull/119/commits/39cc70a8e57e77e23f062f14b860863e119bcb9a.